### PR TITLE
[zh] Resync workloads concepts page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -1182,25 +1178,35 @@ The tradeoffs are:
...
+  A single Job for all work items is better for large numbers of items. 
+- Number of Pods created equals number of work items, versus each Pod can process multiple work items.
+  When the number of Pods equals the number of work items, the Pods typically
+  requires less modification to existing code and containers. Having each Pod
+  process multiple work items is better for large numbers of items.
+-->
+- 每个工作条目对应一个 Job 或者所有工作条目对应同一 Job 对象。
+  为每个工作条目创建一个 Job 的做法会给用户带来一些额外的负担，系统需要管理大量的 Job 对象。
+  用一个 Job 对象来完成所有工作条目的做法更适合处理大量工作条目的场景；
+- 创建数目与与工作条目相等的 Pod 或者令每个 Pod 可以处理多个工作条目。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1193]:
```suggestion
- 创建数目与工作条目相等的 Pod 或者令每个 Pod 可以处理多个工作条目。
```

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -1182,25 +1178,35 @@ The tradeoffs are:
...
+  large numbers of Job objects.
+  A single Job for all work items is better for large numbers of items. 
+- Number of Pods created equals number of work items, versus each Pod can process multiple work items.
+  When the number of Pods equals the number of work items, the Pods typically
+  requires less modification to existing code and containers. Having each Pod
+  process multiple work items is better for large numbers of items.
+-->
+- 每个工作条目对应一个 Job 或者所有工作条目对应同一 Job 对象。
+  为每个工作条目创建一个 Job 的做法会给用户带来一些额外的负担，系统需要管理大量的 Job 对象。
+  用一个 Job 对象来完成所有工作条目的做法更适合处理大量工作条目的场景；
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1192]:
```suggestion
  用一个 Job 对象来完成所有工作条目的做法更适合处理大量工作条目的场景。
```

File: content/zh-cn/docs/concepts/workloads/controllers/statefulset.md [hunk 3]
```diff
@@ -254,12 +254,14 @@ StatefulSet Pod 具有唯一的标识，该标识包括顺序标识、稳定的
...
-pods will be assigned ordinals from 0 up through N-1.
+pods will be assigned ordinals from 0 up through N-1. The StatefulSet controller
+will also add a pod label with this index: `apps.kubernetes.io/pod-index`.
 -->
-### 有序索引   {#ordinal-index}
+### 序号索引   {#ordinal-index}
 
 对于具有 N 个[副本](#replicas)的 StatefulSet，该 StatefulSet 中的每个 Pod 将被分配一个整数序号，
-该序号在此 StatefulSet 上是唯一的。默认情况下，这些 Pod 将被从 0 到 N-1 的序号。
+该序号在此 StatefulSet 中是唯一的。默认情况下，这些 Pod 将赋予被从 0 到 N-1 的序号。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/statefulset.md:263]:
```suggestion
该序号在此 StatefulSet 中是唯一的。默认情况下，这些 Pod 将被赋予从 0 到 N-1 的序号。
```

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 4]
```diff
@@ -2,14 +2,21 @@
 title: "工作负载"
 weight: 55
 description: 理解 Pods，Kubernetes 中可部署的最小计算对象，以及辅助它运行它们的高层抽象对象。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/_index.md:4]:
```suggestion
description: 理解 Kubernetes 中可部署的最小计算对象 Pod 以及辅助运行这些 Pod 的上层抽象。
```
Seems this desc need to be improved too.

**Review by windsonsea — COMMENTED** [review]
Thanks.
/approve

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 5]
```diff
@@ -60,65 +62,71 @@ Kubernetes provides several built-in workload resources:
...
+  Pod 都是相互等价的，并且在需要的时候被替换。
 * [StatefulSet](/zh-cn/docs/concepts/workloads/controllers/statefulset/)
   让你能够运行一个或者多个以某种方式跟踪应用状态的 Pod。
-  例如，如果你的负载会将数据作持久存储，你可以运行一个 `StatefulSet`，将每个
-  `Pod` 与某个 [`PersistentVolume`](/zh-cn/docs/concepts/storage/persistent-volumes/)
-  对应起来。你在 `StatefulSet` 中各个 `Pod` 内运行的代码可以将数据复制到同一
-  `StatefulSet` 中的其它 `Pod` 中以提高整体的服务可靠性。
+  例如，如果你的负载会将数据作持久存储，你可以运行一个 StatefulSet，将每个
+  Pod 与某个 [PersistentVolume](/zh-cn/docs/concepts/storage/persistent-volumes/)
+  对应起来。你在 StatefulSet 中各个 `Pod` 内运行的代码可以将数据复制到同一
```
Comment (my-git9) [inline content/zh-cn/docs/concepts/workloads/_index.md:86]:
```suggestion
  对应起来。你在 StatefulSet 中各个 Pod 内运行的代码可以将数据复制到同一
```

**Review by my-git9 — COMMENTED** [review]
/lgtm

**Comment by windsonsea** [comment]
The CI has a problem. Need to trigger it again.

**Pull request merged**